{{cookiecutter.project_name}}/tests/test_token.py

```diff
@@ -185,11 +185,33 @@ def test_mint(token, owner, receiver):
     assert logs[0].receiver == receiver.address
     assert logs[0].amount == 420
 
-    receiver_balance = token.balanceOf(receiver) 
+    receiver_balance = token.balanceOf(receiver)
     assert receiver_balance == 420
 
     totalSupply = token.totalSupply()
     assert totalSupply == 1420
+
+
+def test_add_minter(token, owner, accounts):
+    """
+    Test adding new minter.
+    Must trigger MinterAdded Event.
+    Must return true when checking if target isMinter
+    """
```

Comment:
I usually like to assert the pre condition too, e.g. That it's not set before you set it

Reply:
```suggestion
    """
    assert token.isMinter(target) is False
```